feature-libs/user/profile/assets/translations/en/userProfile.json

```diff
@@ -30,7 +30,6 @@
     "confirmNewPassword": "Confirm New Password",
     "resetPassword": "Reset Password",
     "createAccount": "Create an account",
-    "title": "Title",
```

Comment:
I'm not sure about removing this translation key as it would constitute a breaking change. Can you confirm if the key has only been introduced to set the aria-label on ng-select? And nowhere else in the past?

Reply:
Hi @Zeyber! I have checked in the repository and couldn't find such combination other than in a place it was removed from. 
Please let me know if I need to revert this change. Thanks a lot 👍